src/app/components/home/home.component.html

```diff
@@ -1,38 +1,52 @@
-@if (trips$ | async; as trips) {
-  <div class="flex flex-col items-center overflow-x-hidden p-4">
-    @for (trip of trips; track trip.docId) {
-      <div class="boundary relative flex w-[150%] justify-center">
-        <div
-          class="absolute z-[-1] flex h-calc w-[60%] overflow-hidden rounded-xl">
-          <div class="w-1/2 bg-green-500"></div>
-          <div class="w-1/2 bg-red-500"></div>
-        </div>
-        <div
-          swipe
-          (left)="onSwipeLeft(trip.title)"
-          (right)="onSwipeRight(trip.title)"
-          (onClick)="handleTripClick(trip.title, trip.docId)"
-          class="w-3/5"
-          cdkDrag
-          cdkDragLockAxis="x"
-          (cdkDragReleased)="resetPosition($event)"
-          cdkDragBoundary=".boundary">
-          @if (
-            trip.startDate && trip.endDate && trip.duration && trip.totalCost
-          ) {
-            <app-trip-card
-              [tripTitle]="trip.title"
-              [startDate]="trip.startDate"
-              [endDate]="trip.endDate"
-              [duration]="trip.duration"
-              [totalCost]="trip.totalCost" />
-          }
-        </div>
+@if ((loading$ | async) ?? false) {
+  <div>Loading...</div>
+} @else {
+  @if (trips$ | async; as trips) {
+    @if (trips.length > 0) {
+      <div class="flex flex-col items-center overflow-x-hidden p-4">
+        @for (trip of trips; track trip.docId) {
+          <div class="boundary relative flex w-[150%] justify-center">
+            <div
+              class="absolute z-[-1] flex h-calc w-3/5 overflow-hidden rounded-xl">
+              <div class="w-1/2 bg-green-500"></div>
+              <div class="w-1/2 bg-red-500"></div>
+            </div>
+            <div
+              swipe
+              (left)="onSwipeLeft(trip.title)"
+              (right)="onSwipeRight(trip.title)"
+              (onClick)="handleTripClick(trip.title, trip.docId)"
+              class="w-3/5"
+              cdkDrag
+              cdkDragLockAxis="x"
+              (cdkDragReleased)="resetPosition($event)"
+              cdkDragBoundary=".boundary">
```

Comment:
What's the dot prefix for?

Reply:
https://material.angular.io/cdk/drag-drop/overview#cdk-drag-drop-boundary